server/routes/creatingReports/checkYourAnswers.ts

```diff
@@ -65,4 +65,8 @@ export default class CheckAnswerRoutes {
     const location = reportId ? `/${reportId}/report-sent` : `/`
     return res.redirect(location)
   }
+
+  public viewReportSent = (req: Request, res: Response): void => {
+    return res.render('pages/report-sent', { data: res.locals.formObject, reportId: req.params.reportId })
+  }
```

Comment:
It was really easy splitting the routes into these three areas which reflects that the division already existed to some extent in the code. 

This was the only route that had to be moved it, it was previously in the `incidents.ts` set of routes. Seemed to make sense that this moved to `creatingReports` as  this is the final step of creating a route - viewing the confirmation page